### PR TITLE
SC-17546 add release notes for sematext-agent 3.2.0

### PR DESCRIPTION
**Pull request by boratanrikulu**

### CODE REVIEW

File: docs/agents/sematext-agent/releasenotes.md [hunk 1]
```diff
@@ -35,6 +39,60 @@ NULL
...
+
+- CronJobs Monitoring: We are happy to introduce metrics collection for CronJobs. Now, you can gain insights into the metrics of your CronJobs. You can check all supported CronJob metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#cronjob-overview-metrics).
+- Kube-controller Monitoring: We now support the collection of metrics from kube-controller, providing you with valuable data to monitor Kubernetes controller performance. You can check all supported kube-controller metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#kube-controller-metrics).
+- Kubernetes Pod Labels: We have added support for collecting metrics related to Kubernetes Pod Labels, allowing for more in-depth monitoring of your pods.
+- MySQL General Logs Support: In this update, we introduce support for parsing MySQL general logs.
+
+### Improvements
+
+- API Server: We now monitor the API server even if the nodes are unreachable.
+- Ship Heartbeat Failures: With this improvement, we ensure that heartbeat failures are properly handled and reported.
```
Comment (boratanrikulu) [inline docs/agents/sematext-agent/releasenotes.md:56]: @yago-123

File: docs/agents/sematext-agent/releasenotes.md [hunk 2]
```diff
@@ -45,14 +45,14 @@ Date: October 19, 2023
 
 ### New Features
 
-- CronJobs Metrics: We are happy to introduce metrics collection for CronJobs. Now, you can gain insights into the performance of your CronJobs.
+- CronJob Metrics: We are happy to introduce metrics collection for CronJob. Now, you can gain insights into the metrics of your CronJobs. You can check all supported CronJoc metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#cronjob-overview-metrics).
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md]: > Kubernetes CronJobs: New CronJobs metrics added, for a complete list, please refer to our [Kubernetes Monitoring Integration docs](https://sematext.com/docs/integration/kubernetes/#cronjob-overview-metrics).

File: docs/agents/sematext-agent/releasenotes.md [hunk 3]
```diff
@@ -35,6 +39,60 @@ NULL
...
 -->
 
+## Version 3.2.0
+
+Date: October 19, 2023
+
+### New Features
+
+- Kubernetes CronJobs monitoring: We introduce metrics collection for Kubernetes CronJobs. You can check all supported CronJob metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#cronjob-overview-metrics).
+- Kube-controller monitoring: We now support the collection of metrics from kube-controller. You can check all supported kube-controller metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#kube-controller-metrics).
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:49]: Kube-controller monitoring -> Kubernetes Controller Manager monitoring:

**Review by yago-123 — APPROVED** [review]
LGTM

File: docs/agents/sematext-agent/releasenotes.md (hunk 1 above)
Comment (yago-123) [reply docs/agents/sematext-agent/releasenotes.md]: Ship heartbeat failures is from Etcd, we added a metric from Etcd that represents the heartbeat failures for that component

**Review by cpipilas — APPROVED** [review]
LGTM

**Pull request merged**